### PR TITLE
Added simple help description to the client CLI

### PR DESCRIPTION
**Pull request by JohannesNicholas**
Hi, 
I made some mistakes with the previous pull request https://github.com/filesender/filesender/pull/1412 
(I didn't realize it would keep up to date with my latest commits to my own fork.)

 The following changes change the output help output of the command line. 

Running `python filesender.py --help` currently will output:
```
usage: filesender.py [-h] [-v] [-i] [-p] [-s SUBJECT] [-m MESSAGE] [-g] [--threads THREADS] [--timeout TIMEOUT] [--retries RETRIES] -u USERNAME -a APIKEY -r RECIPIENTS files [files ...]

positional arguments:
  files                 path to file(s) to send

optional arguments:
  -h, --help            show this help message and exit
  -v, --verbose
  -i, --insecure
  -p, --progress
  -s SUBJECT, --subject SUBJECT
  -m MESSAGE, --message MESSAGE
  -g, --guest
  --threads THREADS
  --timeout TIMEOUT
  --retries RETRIES

required named arguments:
  -u USERNAME, --username USERNAME
  -a APIKEY, --apikey APIKEY
  -r RECIPIENTS, --recipients RECIPIENTS
```
  
  With these changes, it outputs:
 ```
python filesender.py --help
usage: filesender.py [-h] [-v] [-i] [-p] [-s SUBJECT] [-m MESSAGE] [-g] [--threads THREADS] [--timeout TIMEOUT] [--retries RETRIES]
                     [-u USERNAME] [-a APIKEY] -r RECIPIENTS
                     files [files ...]

File Sender CLI client.
Source code: https://github.com/filesender/filesender/blob/master/scripts/client/filesender.py

positional arguments:
  files                 path to file(s) to send

options:
  -h, --help            show this help message and exit
  -v, --verbose
  -i, --insecure
  -p, --progress
  -s SUBJECT, --subject SUBJECT
  -m MESSAGE, --message MESSAGE
  -g, --guest
  --threads THREADS
  --timeout TIMEOUT
  --retries RETRIES
  -u USERNAME, --username USERNAME
  -a APIKEY, --apikey APIKEY

required named arguments:
  -r RECIPIENTS, --recipients RECIPIENTS

A config file can be added to C:\Users\Joey/.filesender/filesender.py.ini to avoid having to specify username and apikey on the command line.

Example (Config file is present): 
python filesender.py -r reciever@example.com file1.txt
```

### CODE REVIEW

**Comment by madsi1m** [comment]
C:\Users\Joey/ ??

File: scripts/client/filesender.py [hunk 1]
```diff
@@ -80,7 +81,18 @@
...
 #argv
-parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.RawDescriptionHelpFormatter,
+    description=textwrap.dedent(f'''\
+      File Sender CLI client.
+      Source code: https://github.com/filesender/filesender/blob/master/scripts/client/filesender.py
+    '''),
+    epilog=textwrap.dedent(f'''\
+      A config file can be added to {homepath}/.filesender/filesender.py.ini to avoid having to specify username and apikey on the command line.
```
Comment (JohannesNicholas) [inline scripts/client/filesender.py:91]: home path is inserted here

**Comment by JohannesNicholas** [comment]
Yes, C:\Users\Joey is my home directory, yours will show up differently, see https://github.com/filesender/filesender/pull/1413/files#r1150366687

**Comment by monkeyiq** [comment]
In the with these changes output I see that 'optional arguments' are listed twice?

**Comment by JohannesNicholas** [comment]
My mistake, it must have been an error when copying the text over from the terminal. I've updated the description to include the actual output.

**Pull request merged**